packages/code-snippet-block/src/components/CodeMirrorEditor/codeMirrorEditor.css

```diff
@@ -0,0 +1,41 @@
+.cm-editor {
+    border: var(--editor-border);
+    padding: var(--editor-padding);
+    border-radius: var(--editor-border-radius);
+}
+
+.cm-panels-top {
+    border-radius: var(--editor-border-top-left-radius) var(--editor-border-top-right-radius) 0 0;
+}
+
+.cm-gutters {
+    border-radius: 0 0 0 var(--editor-border-bottom-left-radius);
+    font-family: Courier;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 16px;
+    letter-spacing: 0.02em;
+    text-align: center;
+}
+
+.ͼ1.cm-editor.cm-focused {
```

Comment:
That is a weird `c` 😄

Reply:
It is an internal class of the editor.

Reply:
Oh, you mean how it looks. Yeah, super weird. :D